<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrimerSearchParametersInfoDialog</class>
 <widget class="QDialog" name="PrimerSearchParametersInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Primer Search Parameters</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../resources.qrc">
    <normaloff>:/resources/images/icons/seanau/Information-B.png</normaloff>:/resources/images/icons/seanau/Information-B.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QPushButton" name="startWizardButton_">
     <property name="text">
      <string>Open &amp;wizard with these settings</string>
     </property>
     <property name="icon">
      <iconset resource="../../../resources.qrc">
       <normaloff>:/resources/images/icons/oxygen/actions/tools-wizard.png</normaloff>:/resources/images/icons/oxygen/actions/tools-wizard.png</iconset>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="verticalSpacing">
      <number>12</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Amplicon Size Range (bp):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="ampliconSizeRangeLabel_">
       <property name="text">
        <string>## .. ###</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Primer Length Range (bp):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="primerLengthRangeLabel_">
       <property name="text">
        <string>## .. ###</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>T&lt;sub&gt;m&lt;/sub&gt; Range (Â°C):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="tmRangeLabel_">
       <property name="text">
        <string>##.# .. ##.#</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="maximumTmStaticLabel_">
       <property name="text">
        <string>Maximum Tm (FIX):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="maximumTmLabel_">
       <property name="text">
        <string>#.#</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>[Na&lt;sup&gt;+&lt;/sup&gt;] (mM):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="sodiumConcentrationLabel_">
       <property name="text">
        <string>###</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>[Primer DNA] (C&lt;sub&gt;T&lt;/sub&gt; uM):</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="primerDnaConcentrationLabel_">
       <property name="text">
        <string>###</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>5' Additions:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Forward:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="forwardReLabel_">
        <property name="text">
         <string>NNNNN (RE name)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Reverse:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="reverseReLabel_">
        <property name="text">
         <string>NNNNN (RE name)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>3' Specifications:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Forward:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="forwardTerminalLabel_">
        <property name="text">
         <string>N*N</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Reverse:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="reverseTerminalLabel_">
        <property name="text">
         <string>N*N</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrimerSearchParametersInfoDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>418</y>
    </hint>
    <hint type="destinationlabel">
     <x>182</x>
     <y>421</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startWizardButton_</sender>
   <signal>released()</signal>
   <receiver>PrimerSearchParametersInfoDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>25</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
